syntax="proto3";
package clusterpb;
option go_package = "./;clusterpb";

message HandshakeRequest {
    string  NodeName = 1;
    string  Host = 2;
    int32   Port = 3;
    int64   Version = 4;
}

message HandshakeResponse {
    int32   Code = 1;
}

message HandshakeAck {
 
}

message GateInstead {
    uint64 userId = 1;
}
